components/reports/precipitation/ReportPrecipIndicatorsTable.vue

```diff
@@ -0,0 +1,425 @@
+          </td>
+          <td>
+            {{ reportData['cwd']['longterm']['MRI-CGCM3']['rcp85']['mean'] }}
+          </td>
+          <td>
+            {{ reportData['cwd']['longterm']['NCAR-CCSM4']['rcp85']['mean'] }}
+          </td>
+        </tr>
+        <tr>
+          <th scope="row">Consecutive Dry Days</th>
+          <td class="left">
+            {{
+              reportData['cdd']['historical']['Daymet']['historical']['mean']
+            }}
+          </td>
+          <td>
+            {{ reportData['cdd']['midcentury']['MRI-CGCM3']['rcp45']['mean'] }}
+          </td>
+          <td>
+            {{ reportData['cdd']['midcentury']['NCAR-CCSM4']['rcp45']['mean'] }}
+          </td>
+          <td>
+            {{ reportData['cdd']['midcentury']['MRI-CGCM3']['rcp85']['mean'] }}
+          </td>
+          <td>
+            {{ reportData['cdd']['midcentury']['NCAR-CCSM4']['rcp85']['mean'] }}
+          </td>
+          <td>
+            {{ reportData['cdd']['longterm']['MRI-CGCM3']['rcp45']['mean'] }}
+          </td>
+          <td>
+            {{ reportData['cdd']['longterm']['NCAR-CCSM4']['rcp45']['mean'] }}
+          </td>
+          <td>
+            {{ reportData['cdd']['longterm']['MRI-CGCM3']['rcp85']['mean'] }}
+          </td>
+          <td>
+            {{ reportData['cdd']['longterm']['NCAR-CCSM4']['rcp85']['mean'] }}
+          </td>
+        </tr>
+        <tr>
+          <th scope="row">Heavy precipitation days <span class="description">1-day precipitation &ge;0.4in</span></th>
+          <td class="left">
+            {{
+              reportData['r10mm']['historical']['Daymet']['historical']['mean']
+            }}
+          </td>
+          <td>
+            {{
+              reportData['r10mm']['midcentury']['MRI-CGCM3']['rcp45']['mean']
+            }}
+          </td>
+          <td>
+            {{
+              reportData['r10mm']['midcentury']['NCAR-CCSM4']['rcp45']['mean']
+            }}
+          </td>
+          <td>
+            {{
+              reportData['r10mm']['midcentury']['MRI-CGCM3']['rcp85']['mean']
+            }}
+          </td>
+          <td>
+            {{
+              reportData['r10mm']['midcentury']['NCAR-CCSM4']['rcp85']['mean']
+            }}
+          </td>
+          <td>
+            {{ reportData['r10mm']['longterm']['MRI-CGCM3']['rcp45']['mean'] }}
+          </td>
+          <td>
+            {{ reportData['r10mm']['longterm']['NCAR-CCSM4']['rcp45']['mean'] }}
+          </td>
+          <td>
+            {{ reportData['r10mm']['longterm']['MRI-CGCM3']['rcp85']['mean'] }}
+          </td>
+          <td>
+            {{ reportData['r10mm']['longterm']['NCAR-CCSM4']['rcp85']['mean'] }}
+          </td>
+        </tr>
+      </tbody>
+      <tfoot>
+        <tr>
+          <td colspan="10" class="about">
+            This table lists some climate indicators derived from a wide range
+            of models.
+            <nuxt-link :to="{ name: 'about' }"
```

Comment:
We probably will need a special section for the indicators dataset explainer but we can wait until we're getting more broad feedback (from our internal team).